renovate.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "automerge": false,
+  "commitMessageAction": "Bump",
+  "commitMessageExtra": "to {{newValue}}",
+  "commitMessageTopic": "{{depName}}",
+  "dependencyDashboard": true,
+  "rebaseWhen": "conflicted",
+  "ignoreDeps": [],
+  "labels": ["dependencies"],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": "before 6:00am on the first day of the month"
+  },
+  "packageRules": [
+    {
+      "matchDepTypes": ["peerDependencies"],
+      "rangeStrategy": "widen"
+    },
+    {
+      "groupName": "babel",
+      "matchPackagePatterns": ["^@babel/", "^@types/babel"]
+    },
+    {
+      "groupName": "Emotion",
+      "matchPackagePatterns": "@emotion/*"
+    },
+    {
+      "groupName": "WyW-in-JS",
+      "matchPackagePatterns": ["@wyw-in-js/*"]
+    },
+    {
+      "groupName": "Font awesome SVG icons",
+      "matchPackagePatterns": "@fortawesome/*"
+    },
+    {
+      "groupName": "core-js",
+      "matchPackageNames": ["core-js"],
+      "allowedVersions": "< 2.0.0"
+    },
+    {
+      "groupName": "JSS",
+      "matchPackageNames": [
+        "css-jss",
+        "jss-plugin-cache",
+        "jss-plugin-camel-case",
+        "jss-plugin-compose",
+        "jss-plugin-default-unit",
+        "jss-plugin-expand",
+        "jss-plugin-extend",
+        "jss-plugin-global",
+        "jss-plugin-isolate",
+        "jss-plugin-nested",
+        "jss-plugin-props-sort",
+        "jss-plugin-rule-value-function",
+        "jss-plugin-rule-value-observable",
+        "jss-plugin-template",
+        "jss-plugin-vendor-prefixer",
+        "jss-preset-default",
+        "jss-starter-kit",
+        "jss",
+        "react-jss"
+      ]
+    },
+    {
+      "groupName": "MUI X",
```

Comment:
You may want to update all Core packages at the same time, similarly to how Base UI does it.

Reply:
Nice catch, I will replace this one with MUI, I don't think we will be showing MUI X components in this docs (at least till we do, no need to have it here).